clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerProtocol.java

```diff
@@ -16,6 +16,9 @@
  */
 package org.apache.kafka.clients.consumer.internals;
 
+import java.util.Collections;
```

Comment:
Just a suggestion here: I feel we should consider migrating this protocol to be auto generated to easier adapt in long term. cc @guozhangwang @hachikuji